src/swarm/neo/client/mixins/RequestCore.d

```diff
@@ -190,13 +195,28 @@ public template RequestCore ( RequestType request_type_, ubyte request_code,
 
     public static struct Context
     {
+        import ocean.util.serialize.contiguous.Serializer;
+        import ocean.util.serialize.contiguous.Deserializer;
+
+        private void[] user_specified_params_blob;
```

Comment:
>the host struct is reused, right?

A `Context` instance is constructed and serialized. Each request handler is passed the serialized buffer and deserializes a `Context` struct again.